### ISSUE TITLE
Add links to WCAG documentation

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-em-report-tool#46]
Links to WCAG documentation is both useful for the auditor and for in the final report. These should be added.

**Comment by WilcoFiers** [comment]
Suggestion added as of #263, needs review.

**Comment by davidberman** [comment]
"Key Resources" offers this to the report tool user, however I think the idea here is that there would be an option to include the short version of each SC within the resulting report document.
